labgrid/util/ssh.py

```diff
@@ -191,23 +192,75 @@ def wrapper(self, *_args, **_kwargs):
         return wrapper
 
     @_check_connected
-    def run_command(self, command):
+    def run(self, command, *, codec = "utf-8", decodeerrors = "strict",
+            timeout = None): # pylint: disable=unused-argument
+
         """Run a command over the SSHConnection
 
         Args:
             command (string): The command to run
+            codec (string, optional): output encoding. Defaults to "utf-8".
+            decodeerrors (string, optional): behavior on decode errors. Defaults
+                 to "strict". Refer to stdtypes' bytes.decode for details.
+            timeout (float, optional): currently unused
```

Comment:
Please don't add unused arguments.

Reply:
Removed. I had included them for compatibility with SSHDriver.